go-controller/pkg/ovn/address_set/address_set.go

```diff
@@ -118,10 +118,10 @@ func ensureOvnAddressSet(nbClient libovsdbclient.Client, name string) (*ovnAddre
 			Op:      ovsdb.OperationWait,
 			Timeout: &timeout,
 			Table:   "Address_Set",
-			Where:   []ovsdb.Condition{{Column: "name", Function: ovsdb.ConditionEqual, Value: name}},
+			Where:   []ovsdb.Condition{{Column: "name", Function: ovsdb.ConditionEqual, Value: as.hashName}},
```

Comment:
Would it make sense to just use `addSet.Name` and avoid the problem altogether?

Reply:
@dcbw agreed and done.